### PR TITLE
Added Stanza snippet

### PR DESCRIPTION
**Pull request by merveenoyan**

### CODE REVIEW

**Comment by merveenoyan** [comment]
Please do not review now :)

File: js/src/lib/interfaces/Libraries.ts [hunk 1]
```diff
@@ -239,6 +239,14 @@ nlp = spacy.load("${nameWithoutNamespace(model.id)}")
 import ${nameWithoutNamespace(model.id)}
 nlp = ${nameWithoutNamespace(model.id)}.load()`;
 
+const stanza = (model: ModelData) =>
+	`from stanza import Pipeline
+
+stanza.download("${nameWithoutNamespace(model.id)}")
+nlp = stanza.Pipeline("${nameWithoutNamespace(model.id)}")
```
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:246]: In stanza you just need the language, so you can also remove `stanza-` from the id. 

```
import stanza
stanza.download('en')
nlp = stanza.Pipeline('en')

File: js/src/lib/interfaces/Libraries.ts [hunk 2]
```diff
@@ -239,6 +239,14 @@ nlp = spacy.load("${nameWithoutNamespace(model.id)}")
 import ${nameWithoutNamespace(model.id)}
 nlp = ${nameWithoutNamespace(model.id)}.load()`;
 
+const stanza = (model: ModelData) =>
+	`from stanza import Pipeline
+
+stanza.download("${nameWithoutNamespace(model.id)}")
+nlp = stanza.Pipeline("${nameWithoutNamespace(model.id)}") 
+doc = nlp("My name is Omar and I live in Zürich.")`;
```
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:247]: I think we should remove this as the input is dependant on the language

File: js/src/lib/interfaces/Libraries.ts [hunk 3]
```diff
@@ -426,6 +434,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: { [key in keyof typeof ModelLibrary]?:
 		repoUrl:  "https://github.com/speechbrain/speechbrain",
 		snippet:  speechbrain,
 	},
+	"stanza": {
+		btnLabel: "stanza",
```
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:438]:
```suggestion
		btnLabel: "Stanza",
```

**Review by osanseviero — CHANGES_REQUESTED** [review]
Thanks for the PR! :fire: :fire: :fire: 

There are a couple of things we need to change for this to work :smile:

**Comment by merveenoyan** [comment]
latest-est version:
Ekran Resmi 2022-06-03 15 36 23

File: js/src/lib/interfaces/Libraries.ts (hunk 1 above)
Comment (merveenoyan) [reply js/src/lib/interfaces/Libraries.ts]: see below :)

File: js/src/lib/interfaces/Libraries.ts [hunk 4]
```diff
@@ -239,6 +239,13 @@ nlp = spacy.load("${nameWithoutNamespace(model.id)}")
 import ${nameWithoutNamespace(model.id)}
 nlp = ${nameWithoutNamespace(model.id)}.load()`;
 
+const stanza = (model: ModelData) =>
+	`from stanza import Pipeline
```
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:243]: The current import will not work for lines 245 and 256 which use `stanza`
```suggestion
	`import stanza
```

**Review by osanseviero — APPROVED** [review]
Left a final comment but otherwise looks good! :fire:

**Comment by julien-c** [comment]
very cool!

**Pull request merged**